src/styles/components/_header.scss

```diff
@@ -30,4 +34,8 @@ $breakpoint-tablet: 820px;
   @media only screen and (min-width: 100rem) {
     margin: auto;
   }
+
+  @media screen and (prefers-color-scheme: dark) {
+    filter: invert(100);
```

Comment:
This dark-mode query should be deleted. Because of the cascade, this query at line 25 is inverting the Sparkbox logo, making it white. ✅ But then with the duplicated dark-mode query, inverting it again, it's turning it back to black. 😩

Reply:
I am going to check this out...but I think we need to just delete the Sparkbox logo altogether based on the design meeting we had. I'll discuss this w/ Dustin at the  pairing we have in a bit.

Reply:
I fixed the logo--I think deleting it would be out of scope here! I also did the feat-dark-mode rebase.  Please re-review when you get a chance! :)

![image](https://user-images.githubusercontent.com/49820744/201190218-88be189b-3f7d-4194-beac-f976648cb773.png)